projects/core/src/lib/datasets/add-data/add-data.component.html

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
Icon button not centered 

![image](https://user-images.githubusercontent.com/1027102/210560539-c1f52f0d-c2b8-4221-8f2b-12101c0f07d5.png)

Reply:
[bc3f138](https://github.com/geo-engine/geoengine-ui/pull/555/commits/bc3f1384584cc0cdfc93122f587c27d7a6325e60)